setup.sh

```
@@ -45,18 +45,20 @@ do
   done
 done
 
-for file in ".tmux.conf" ".vimrc" ".zshrc" ".zshenv" ".zprofile"
+for file in ".tmux.conf" ".zshrc" ".zshenv" ".zprofile"
 do
   src="$DOTPATH/$file"
   ln -snfv $src $HOME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)